rosdep/base.yaml

```
@@ -8469,11 +8469,16 @@ rti-connext-dds-5.3.1:
   ubuntu:
     bionic: [rti-connext-dds-5.3.1]
     focal: [rti-connext-dds-5.3.1]
-rti-connext-dds-6.0.1:
+rti-connext-dds-6.0.1:  # Supported until ubuntu 24.04
   nixos: []
   ubuntu:
-    '*': [rti-connext-dds-6.0.1]
-    bionic: null
+    focal: [rti-connext-dds-6.0.1]
+    jammy: [rti-connext-dds-6.0.1]
+    noble: [rti-connext-dds-6.0.1]
+rti-connext-dds-7.3.0-ros:  # supported from ubuntu 24.04 noble
+  nixos: []
+  ubuntu:
+    noble: [rti-connext-dds-7.3.0-ros]
```

In order to maintain forward compatibility we usually invert the definition here:
```suggestion
  ubuntu:
    '*': [rti-connext-dds-7.3.0-ros]
    jammy: null
    focal: null
```

This will allow us to use this package in 26.04 without requiring re-definition. The method by which we currently populate bootstrap repositories will insure this package is propagated to 26.04 when we first create it. However, if the Connext maintainers prefer explicitly defining which distros are supported for each rosdep definition I'm alright with that.

---

@nuclearsandwich Just a quick question to avoid any issues—can you confirm whether I should use `rti-connext-dds-7.3.0` instead of `rti-connext-dds-7.3.0-ros` in [this PR](https://github.com/ros2/rmw_connextdds/pull/181) in `rti_connext_dds_cmake_module/package.xml`?